rpc-tests/tests/test-dapp-staking.js

```diff
@@ -0,0 +1,27 @@
+import { expect } from 'chai';
+import { Keyring } from '@polkadot/api';
+import { describeWithAstar, wait } from './util.js';
+
+const CONTRACT = '0x000000000000000000000000000000000000000001'; //0x01
```

Comment:
Please use concrete types/interfaces.
Hardcoding encoded values isn't the way to go, especially not since this is the introduction of this code, not some legacy.

As suggested in Slack, check the `astar-apps` to see how they do it there.

Reply:
This is just a precompiled contract which is used to test register functionallity.

astar-apps is also using precompiled contract values https://github.com/AstarNetwork/astar-apps/blob/b964c05f1c25ad96cdd0e5d663e9cdae1c97556b/src/hooks/useStaking.ts#L12

![Screenshot from 2022-07-07 12-42-02](https://user-images.githubusercontent.com/874046/177755301-af55a10e-b1c3-44aa-927c-d187bc7c2e3d.png)